src/ECS/Systems/Systems.cpp

```diff
@@ -256,6 +256,51 @@ namespace Systems {
         }
     }
 
+    constexpr int outsideWindowTopLeft  = 0;
+    constexpr int outsideWindowBotRigth = 100;
+
+    static bool isOutsideWindow(const Types::Position &position)
+    {
+        if (position.x < outsideWindowTopLeft || position.x > outsideWindowBotRigth
+            || position.y < outsideWindowTopLeft || position.y > outsideWindowBotRigth) {
+            return (true);
+        }
+        return (false);
+    }
+
+    void manageOutsideWindowEntity(std::size_t /*unused*/, std::size_t /*unused*/)
+    {
+        Registry::components<Types::Position> arrPosition =
+            Registry::getInstance().getComponents<Types::Position>();
+        Registry::components<Types::Parallax> arrParallax =
+            Registry::getInstance().getComponents<Types::Parallax>();
+        Registry::components<Types::Dead> arrDead = Registry::getInstance().getComponents<Types::Dead>();
+
+        std::vector<std::size_t> ids = arrPosition.getExistingsId();
+
+        for (auto &id : ids) {
+            if (!arrParallax.exist(id)) {
+                if (isOutsideWindow(arrPosition[id])) {
+                    executeDeathFunction(id, arrDead);
```

Comment:
As the entity is no longer visible on the screen, we might as well call DestroyEntity directly as the Death function. Who cares about anims as we're not going to see them?